### ISSUE TITLE
SCSS wont compile on windows

### ISSUE THREAD

**Issue by TylersDesk** [zackphilipps/scratch-theme#8]
Installed gulp, grunt, node-sass.
Ran NPM install from root.

Run either `gulp style`s or `grunt sass` and produces the following error.

Missing an @import statement?

```
Running "sass:style" (sass) task
>> Invalid CSS after ".": expected 1 selector or at-rule, was "../../../node_modul"
>>   Line 1  Column 1  ..\core\scss\_ionicons.scss
```

I've never seen a scss file start with a path to another file, that is the case with _ionicons.scss:
```
../../../node_modules/ionicons/css/ionicons.css
```

**Comment by TylersDesk** [comment]
Happens with all the files that are referencing the node modules..

Is there a step missing in the gulp/grunt tasks to copy files over from the node modules? Is this something that is on the roadmap? I could create the gulp/grunt tasks to fix the issues.

**Comment by zoephilipps** [comment]
@TylersDesk 👋 

I believe this is an issue with symlinks not being carried over properly... I'll fix that.

**Comment by zoephilipps** [comment]
@TylersDesk This should be fixed in [v1.6.10](https://github.com/zackphilipps/scratch-theme/releases/tag/v1.6.10) (aka latest master). Can you take a look and let me know?
